### PR TITLE
Add sdds package

### PR DESCRIPTION
**Pull request by fsoubelet**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/sdds/LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (dopplershift) [inline recipes/sdds/LICENSE:1]: This file is already present in the PyPI tarball, so it shouldn't be included here.

**Review by dopplershift — CHANGES_REQUESTED** [review]
Just remove the unneeded license file and this is good to go.

**Comment by fsoubelet** [comment]
Done @dopplershift 

However I'm a little confused as to when to do that, as for instance for https://github.com/conda-forge/staged-recipes/pull/17004 I left the license, which is also included in the tarball.

**Comment by dopplershift** [comment]
@fsoubelet Just means I missed checking for it. Happens when you're reviewing several of these every day. 😉 

So in the feedstock for tfs-pandas you should delete it too.

**Review by dopplershift — APPROVED** [review]

**Comment by fsoubelet** [comment]
Noted, I will take care of that.

**Pull request merged**